crates/aranya-client/src/config.rs

```diff
@@ -0,0 +1,26 @@
+use crate::error::ConfigError;
+
+/// A builder for adding parameters when adding or creating teams.
+#[derive(Clone, Debug, Default, PartialEq, Eq)]
+pub struct TeamConfigBuilder {}
+
+impl TeamConfigBuilder {
+    /// Creates a new builder for [`TeamConfig`]
+    pub fn new() -> Self {
+        Self::default()
+    }
+
+    /// Creates a [`TeamConfig`] using the provided parameters
+    pub fn build(self) -> Result<TeamConfig, ConfigError> {
```

Comment:
unless we flesh out `ConfigError` before MVP, we should probably just make this return `Error`, since it's a superset of `ConfigError`.